boa/profiling.py

```diff
@@ -306,38 +302,45 @@ def get_call_profile_table(env: Env) -> Table:
 
     cache = env._cached_call_profiles
     cached_contracts = env._profiled_contracts
-    contract_vs_mean_gas = []
+    contract_vs_median_gas = []
     for profile in cache:
         cache[profile].compute_stats()
-        contract_vs_mean_gas.append(
-            (cache[profile].net_gas_stats.avg_gas, profile.address)
+        contract_vs_median_gas.append(
+            (cache[profile].net_gas_stats.median_gas, profile.address)
         )
 
     # arrange from most to least expensive contracts:
-    sort_gas = sorted(contract_vs_mean_gas, key=lambda x: x[0], reverse=True)
+    sort_gas = sorted(contract_vs_median_gas, key=lambda x: x[0], reverse=True)
     sorted_addresses = list(set([x[1] for x in sort_gas]))
 
     # --------------- POPULATE TABLE -----------------
 
     for address in sorted_addresses:
-        fn_vs_mean_gas = []
+        fn_vs_median_gas = []
         for profile in cached_contracts[address]:
-            fn_vs_mean_gas.append((cache[profile].net_gas_stats.avg_gas, profile))
+            fn_vs_median_gas.append((cache[profile].net_gas_stats.median_gas, profile))
 
         # arrange from most to least expensive contracts:
-        fn_vs_mean_gas = sorted(fn_vs_mean_gas, key=lambda x: x[0], reverse=True)
+        fn_vs_median_gas = sorted(fn_vs_median_gas, key=lambda x: x[0], reverse=True)
 
-        for c, (_, profile) in enumerate(fn_vs_mean_gas):
+        for c, (_, profile) in enumerate(fn_vs_median_gas):
             stats = cache[profile]
 
             # only first line should be populated for name and address
             cname = ""
-            caddr = ""
-            if c == 0:
-                cname = profile.contract_name
-                caddr = address
             fn_name = profile.fn_name
-            table.add_row(cname, caddr, fn_name, *stats.net_gas_stats.get_str_repr())
+            stats = list(stats.net_gas_stats.get_str_repr())
+            if c == 0:
+                relpath = os.path.relpath(profile.contract_name)
```

Comment:
what if call to relpath fails?

Reply:
it cannot fail as long as contract_name isnt `""`

Reply:
On windows it can fail if the two paths are on different drives
![Screenshot_20240717_093152_Brave](https://github.com/user-attachments/assets/bc1d2b1a-a1a4-4c4a-9c26-819031117cb5)